frontend/src/components/FamilyTree/FamilyTreeVisualizationLazy.tsx

```
@@ -0,0 +1,93 @@
+
+// Error boundary for lazy loading errors
+interface FamilyTreeErrorBoundaryState {
+  hasError: boolean;
+  error?: Error;
+}
+
+class FamilyTreeErrorBoundary extends React.Component<
+  React.PropsWithChildren<{}>,
+  FamilyTreeErrorBoundaryState
+> {
+  constructor(props: React.PropsWithChildren<{}>) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)